improver_tests/cli/test_init.py

```diff
@@ -348,5 +348,13 @@ def test_mismatched_close_brackets(self):
             unbracket(['foo', ']', 'bar'])
 
 
+def test_import_cli():
+    """Test if `import improver.cli` pulls in heavy stuff like numpy."""
+    import subprocess
+    import sys
+    script = 'import improver.cli, sys; assert "numpy" not in sys.modules'
```

Comment:
Suggest adding a comment here explaining why the subprocess is used, as the use to get a fresh import environment is non-obvious for anyone looking at this in the future.
Maybe also worth mentioning that other large libraries commonly used in IMPROVER (eg. iris) will import numpy as a dependency, and that subsequent numpy import will get picked up here.